.travis.yml

```diff
@@ -13,44 +11,42 @@ addons:
     - libvorbis-dev
     - g++
     - clang
+    - freeglut3-dev
```

Comment:
Why did you add freeglut? Did you mean to add freetype?

Reply:
As discussed, it was a stab in the dark based on googling, and we've established libgl1-mesa-dev is better, so that's changed now